specification/transport_layer/can.tex

```diff
@@ -208,16 +207,15 @@ \subsubsection{Source node-ID field in anonymous message transfers}\label{sec:ca
 CAN bus does not allow different nodes to transmit CAN frames with different data field values under the same CAN ID.
 Owing to the fact that the CAN ID includes the node-ID value of the transmitting node,
 this restriction does not affect regular UAVCAN transfers.
-However, anonymous message transfers would violate this restriction,
-because they don't have a unique node-ID.
+However, anonymous message transfers would violate this restriction because they don't have a unique node-ID.
 
 In order to work around this problem,
 UAVCAN requires that the source node-ID field of anonymous messages\footnote{Source node identifier
 is not defined for anonymous message transfers; see table \ref{table:common_transfer_properties}.}
 is initialized with a pseudorandom \emph{pseudo-ID value},
 and defines special logic for handling CAN bus errors during transmission of anonymous frames.
 
-The source of the pseudorandom data used for the pseudo-ID must be likely to produce different values
+The source of the pseudorandom data used for the pseudo-ID must be likely to produce different values % Not sure what 'must be likely' means in this context'
```

Comment:
ugh. we missed a comment

Reply:
:(
Not going to attempt to add a new commit, but found the new text in [this comment](https://github.com/UAVCAN/specification/pull/55#discussion_r330921220): 
"The source of the pseudorandom data used for the pseudo-ID must produce different values for different CAN frame payloads or more anonymous messages will be lost."